src/components/PriorAuth/PriorAuth.jsx

```diff
@@ -131,7 +124,7 @@ export default class PriorAuth extends Component {
   handleGetLink() {
     console.log(window.location.href);
     this.setState({
-      showLink: true
+      showLink: true,
```

Comment:
Why is there a trailing comma?   (I'm not a REACT programmer, so I found this confusing.

Reply:
These were all automatically added by Prettier in vscode. The Prettier plugin auto formats based on the project files so it must be defined somewhere there to make this a rule